pkg/skaffold/inspect/types.go

```diff
@@ -37,6 +37,15 @@ type Options struct {
 	ModulesOptions
 	ProfilesOptions
 	BuildEnvOptions
+	TestsOptions
+}
+
+// TestsOptions holds flag values for various `skaffold inspect tests` commands
+type TestsOptions struct {
+	// Profiles is the slice of profile names to activate.
+	TestsProfiles []string
```

Comment:
Note: name `TestsProfiles` was used (vs `Profiles`) as the name `Profiles` is used in the `BuildEnvs` options which would make `Profiles` ambiguous when used as `opts.Profiles`